crates/iceberg-catalog/src/api/management/mod.rs

```diff
@@ -926,13 +926,12 @@ pub mod v1 {
         )
     )]
     async fn undrop_tabulars<C: Catalog, A: Authorizer + Clone, S: SecretStore>(
-        Path(warehouse_id): Path<uuid::Uuid>,
+        Path(_warehouse_id): Path<uuid::Uuid>,
```

Comment:
why keep this if its unused? Do you want to introduce undrop for a whole warehouse later?

Reply:
I think we should keep it in the path so that we can unrop a table by name later.
Query engines don't know IDs and thus need this translation layer eventually.